tracer/src/Datadog.Trace/SpanContextExtractor.cs

```diff
@@ -80,7 +80,9 @@ internal SpanContextExtractor(bool unusedParamNotToUsePublicApi)
             if (messageType != null && source == null) { ThrowHelper.ThrowArgumentNullException(nameof(source)); }
             else if (messageType == null && source != null) { ThrowHelper.ThrowArgumentNullException(nameof(messageType)); }
 
-            var spanContext = SpanContextPropagator.Instance.Extract(carrier, getter);
+            // TODO: should this silently merge the baggage extracted from the carrier into Baggage.Current?
```

Comment:
TODO: should this (public api) silently merge the baggage extracted from the carrier into Baggage.Current?

Reply:
Yes